envs/scIB-python.yml

```diff
@@ -50,18 +53,16 @@ dependencies:
   - xlrd
   - umap-learn==0.3.10
   - louvain==0.6.1
-  - scvi
+  - scvi==0.6.7
+  - scanorama==1.7.0
   - pip:
-    - anndata==0.7.1
+    - git+git://github.com/theislab/scib.git
     - trvae==1.1.2
     - trvaep==0.1.0
-    - scgen==1.1.5
-    - git+git://github.com/danielStrobl/scanorama@patch-1
     - mnnpy==0.1.9.5
-    - git+git://github.com/theislab/scib.git  
-    - desc==2.0.3
+      # - scgen==1.1.5
+      # - desc==2.0.3
+      # - keras==2.2.4
```

Comment:
could we please keep scgen in the environment? It's desc and scgen that clash, so we can use one over the other. As desc doesn't perform well, let's opt for scgen.

Reply:
This looks like this was mistakenly commented out, but not sure. I think you can install desc first and then scgen after which would install the dependencies for scgen while ignoring the desc ones

Reply:
Okay, could you suggest the fix and then both test locally? I think @mumichae had an issue with this, no?

Reply:
scgen also clashes with trvae
```
The conflict is caused by:
    scgen 1.1.5 depends on keras==2.3
    trvae 1.1.2 depends on keras==2.2.4
```

Reply:
scgen > desc > trvae in importance here atm i think.

Reply:
added scgen and removed trvae